ratelimit/utils.py

```diff
@@ -126,7 +126,13 @@ def is_ratelimited(request, group=None, fn=None, key=None, rate=None,
         return False
     usage = get_usage_count(request, group, fn, key, rate, method, increment)
 
-    limited = usage.get('count') > usage.get('limit')
+    usage_count = usage.get('count')
+    usage_limit = usage.get('limit')
+    limited = \
```

Comment:
Does this make it fail closed or fail open? I feel like we should give some kind of control to users over that behavior.

That would also maybe make it possible to get rid of the `\` line continuations (`(...)` is preferable), e.g.

```py
if usage_count is None or usage_limit is None:
    limited = failure_mode
else:
    limited = usage_count > usage_limit  # avoids calling .get() again, uses local references
```

Is it possible for _one_ of `count` or `limit` to be `None` but not both?

Reply:
It is possible if the cache backend fails or comes back between the two calls. So do you want a new setting to control either fail-open or fail-close?